roles/iommu/meta/requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- name: stackhpc.grubcmdline
```

Comment:
Don't know if this will work :crossed_fingers:

Reply:
Not supported - can only depend on a collection: https://stackoverflow.com/questions/69508170/ansible-role-dependency-handling-for-collections

Reply:
Kind of makes sense as I'm not sure how would know to install it in the roles directory